frontend/packages/integration-tests-cypress/tests/crud/annotations.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { checkErrors, testName } from '../../support';
+import { nav } from '../../views/nav';
+import { listPage } from '../../views/list-page';
+import * as yamlEditor from '../../views/yaml-editor';
+import { errorMessage } from '../../views/form';
+import { detailsPage } from '../../views/details-page';
+import { modal } from '../../views/modal';
+
+const createExampleConfigMapInstance = () => {
+  nav.sidenav.clickNavLink(['Workloads', 'Config Maps']);
+  cy.byLegacyTestID('resource-title').should('have.text', 'Config Maps');
+  listPage.clickCreateYAMLbutton();
+  cy.byTestID('resource-sidebar').should('exist');
+  yamlEditor.isLoaded();
+  yamlEditor.clickSaveCreateButton();
+  cy.get(errorMessage).should('not.exist');
+};
+
+const deleteExampleConfigMapInstance = () => {
+  detailsPage.isLoaded();
+  detailsPage.titleShouldContain('example');
+  detailsPage.clickPageActionFromDropdown('Delete Config Map');
+  modal.shouldBeOpened();
+  modal.submit();
+  modal.shouldBeClosed();
+  cy.get(errorMessage).should('not.exist');
+};
+
+const annotationHelper = (
+  index: number,
+  field: 'key' | 'value',
+  action: 'type' | 'clear' | 'match',
+  value?: string,
+) => {
+  cy.byTestID(field === 'key' ? 'pairs-list-name' : 'pairs-list-value')
+    .eq(index)
+    .then(($el) => {
+      switch (action) {
+        case 'type':
+          cy.wrap($el).type(value);
+          break;
+        case 'clear':
+          cy.wrap($el).clear();
+          break;
+        case 'match':
+          cy.wrap($el).should('have.value', value);
+          break;
+        default:
+        // code block
+      }
+    });
+};
+
+describe('Annotations', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+    createExampleConfigMapInstance();
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    deleteExampleConfigMapInstance();
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  it(`Creates, Edits, Updates, and Deletes Annotations`, () => {
+    const ANNOTATION_2_VALUE = 'ALPHA_Num_VALUE-2';
+    const ANNOTATION_3_KEY = 'ALPHA_Num_KEY-3';
+    const annotations = [
+      {
+        key: 'ALPHA_Num_KEY-1',
+        value: 'ALPHA_Num_VALUE-1',
+      },
+      {
+        key: 'ALPHA_Num_KEY-2',
+        value: '',
+      },
+      {
+        key: '',
+        value: 'ALPHA_Num_VALUE-3',
+      },
+    ];
+
+    cy.url().should('include', `/k8s/ns/${testName}/configmaps/example`);
+    detailsPage.isLoaded();
+    detailsPage.titleShouldContain('example');
+    cy.byLegacyTestID('edit-annotations').contains('0 annotations');
```

Comment:
Is it possible to update the ID not to use the legacy prefix, or is the old test ID used in other places?

Reply:
removed old test id